control_plane/src/background_process.rs

```diff
@@ -0,0 +1,250 @@
+//! Spawns and kills background processes that are needed by Neon CLI.
+//! Applies common set-up such as log and pid files to every process.
+//! Neon CLI does not run in background, so it needs to store the information about
+//! spawned processes which it does in this module.
```

Comment:
Store it where? Why?

Reply:
Added more elaborations on the pid file.